docs/getting_started_guide/Simp_on_AWS.rst

```diff
@@ -0,0 +1,39 @@
+.. _gsg-simp_on_aws:
+
+SIMP on Amazon Web Services
+===========================
+
+This chapter provides notes and guidance on using the official SIMP Amazon
+Machine Image (AMI) to run the SIMP server in the cloud. 
+
+The SIMP AMI is built from the SIMP ISO, so much of the information contained in section 3.5.2 :ref:`ig-simp-server-installation` applies here. 
+
+Provision a New EC2-Instance
+----------------------------
+
+To provision a new ec2-instance in the AWS cloud running the official SIMP AMI, follow these steps:
+   - Launch a new instance in the normal way, and navigate to the AWS Marketplace tab when prompted to choose a Machine Image. 
+   - Search the marketplace for the SIMP AMI, and locate the official published SIMP AMI.
+   - Your ec2-instance should be **at LEAST** ``t2.medium``, with 2 cpus and 4GB of memory. Less than 4GB of memory will significantly slow down the bootstrapping process, and might cause problems in the future. 
```

Comment:
It definitely could. I could link to relevant aws documentation for each bullet, if you think that would be better. I wanted to avoid repeating their documentation as much as possible.